### ISSUE TITLE
Phenotype page: fix links in the "The way we measure" section

### ISSUE THREAD

**Issue by viomunoz** [mpi2/PhenotypeData#557]
**Issue:**
A user has reported that the procedure list in the "The way we measure" section in the Phentoype pages [1] do not redirect to the Procedure description, but to the IMPRESS homepage https://www.mousephenotype.org/impress/index

**Example:** 
https://www.mousephenotype.org/data/phenotypes/MP:0011182#genesAssociations

**Requirement:**
Each Procedure to redirect to the Procedure description in IMPRESS
Example: Haematology to redirect to https://www.mousephenotype.org/impress/ProcedureInfo?action=list&procID=674&pipeID=7

[1] **Screenshot:**
Procedure list in the "The way we measure" section in the Phentoype page "decreased hematopoietic cell number" 
![Screenshot 2021-03-08 at 10.23.39.png](https://images.zenhubusercontent.com/5e271bb2e33c5e6b469a910f/db243946-1cda-4dde-862c-760ff850ffdc)

**Comment by jwgwarren** [comment]
I don't believe there is anything wrong with the code on this section - the links look correct - but maybe they are an old style?
http://www.mousephenotype.org/impress/protocol/1242
this link is for immunophentyping but navigating the impress website using the search you get many links to immunophenotyping in the result with all links now like this: https://www.mousephenotype.org/impress/ProcedureInfo?procID=1241

The links are different for different procedures with different numbers at the end but the page is then being redirected to the impress home page by the proxy or Harwell impress site?

**Comment by jwgwarren** [comment]
Also it seems we put a /4 on the end of the procedure has MGP in the string???? This seems a bit archaich or not very restful???

**Comment by jwgwarren** [comment]
Actually looking at the impress search results the links all go the same place but have different ids.

Changing the links locally on my machine seemed to link to a blank page?? eek??

**Comment by jwgwarren** [comment]
I've removed the special case for MGP and put the new style links in for the hrefs. Now it seems to be working.I'll commit these changes.

**Comment by jwgwarren** [comment]
Create a React version instead - work is in progress.
